src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Solution/FinalizeSolutionStep.cs

```diff
@@ -30,23 +29,15 @@ public FinalizeSolutionStep(ILogger<FinalizeSolutionStep> logger)
         protected override Task<UpgradeStepApplyResult> ApplyImplAsync(IUpgradeContext context, CancellationToken token)
         {
             context.IsComplete = true;
-            context.SetEntryPoint(null);
```

Comment:
I think we still need to clear entry points in this step, otherwise whatever entry point the user chose during this run will be persisted to the state file and they won't be prompted to pick a different entry point next time they run the tool on that solution even though at this point they're done upgrading from that particular entry point.

I think this regresses things so that if a user runs on a sln without -e, even after completing the upgrade, they won't be able to run on that same solution and pick a different entry point.

Reply:
Reverted back, but to updated to check if any endpoints are set